docs/usage.md

```diff
@@ -509,6 +527,18 @@ deepbgc_db/
     └── myDetectors*.pkl
 ```
 
+### InterProScan
+
+[InterProScan](https://github.com/ebi-pf-team/interproscan) is used to provide more information about the proteins annotated on the contigs. By default, turning on this subworkflow with `--run_protein_annotation_interproscan` will download and unzip the (as of now) latest [InterPro database](http://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.67-99.0/) v5.67-99.0. The database downloaded can be saved in the output directory `<output_directors>/databases/interproscan/*` if the `--save_db` is turned on. Note: the download can take upto 4 hours depending on teh bandwidth.
+
+A diifferent version of the database can be supplied to teh pipeline ba passing the InterProScan database directory to `--protein_annotation_interproscan_db path/to/interproscan_db/`. The directory can be created following with:
+
+```
+curl -L https://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.67-99.0/interproscan-5.67-99.0-64-bit.tar.gz -o interproscan_db/interproscan-5.67-99.0-64-bit.tar.gz
+tar -xzf interproscan_db/interproscan-5.67-99.0-64-bit.tar.gz -C interproscan_db/
+
```

Comment:
```suggestion
```

Please also give what files or directory names are expected in the top level of the directory - this helps with making sure a user gives exactly the right directory (in case of nested folders from downloaded archives etc)

Reply:
Its exactly teh untarred directory as mentioed above in `--protein_annotation_interproscan_db path/to/downloaded-untarred-interproscan_db-dir/`